### ISSUE TITLE
Example acquia.config.yml is out-of-date

### ISSUE THREAD

**Issue by timcosgrove** [geerlingguy/drupal-vm#1036]
### Issue Type

 - Documentation Report

### Your Environment

```
Vagrant 1.8.6
VirtualBox 5.1.6r110634
ansible 2.2.0.0
  config file =
  configured module search path = Default w/o overrides
```

### Your OS

 - **macOS** (10.11.6)

### Full console output

Build w/ existing acquia.config.yml: https://gist.github.com/timcosgrove/fa77ebd55e0a2b9437f00e3579bdfbe8
Build w/ modified acquia.config.yml: https://gist.github.com/timcosgrove/58dc8af89ab08f2e8323436a2f847d9d

### Summary

The current acquia.config.yml in examples does not appear to build correctly, at least with my Vagrant/ansible/OS combo.

Steps to reproduce.
1. Check out drupal-vm at master.
2. Copy contents of [acquia.config.yml](https://raw.githubusercontent.com/geerlingguy/drupal-vm/master/examples/acquia/acquia.config.yml) to config.yml at the base of the repo.
3. run `vagrant up`.

Results: see Gist #1 above.

I modified the config based on the [instructions in the PHP5.6 documentation](https://github.com/timcosgrove/drupal-vm/blob/master/docs/other/php-56.md), which appears to fix the problem: see Gist #2.

I'll be submitting a PR with these changes. If this is an actual problem and the PR solves it, enjoy!

Thanks for DVM.

**Comment by timcosgrove** [comment]
https://github.com/geerlingguy/drupal-vm/pull/1037

**Comment by geerlingguy** [comment]
Fixed in #1037 - thanks!
